business/tls.go

```diff
@@ -44,13 +44,15 @@ func (in *TLSService) MeshWidemTLSStatus(ctx context.Context, namespaces []strin
 		IncludePeerAuthentications: true,
 		Cluster:                    cluster,
 	}
+	conf := config.Get()
 
-	istioConfigList, err := in.businessLayer.IstioConfig.GetIstioConfigListPerCluster(ctx, criteria, cluster)
+	// @TODO hardcoded HomeClusterName
+	istioConfigList, err := in.businessLayer.IstioConfig.GetIstioConfigListPerCluster(ctx, criteria, conf.KubernetesConfig.ClusterName)
```

Comment:
I changed this as you suggested.  But I am wondering what is the difference between what is supplied in the `criteria.cluster` and `cluster` arguments?  Also, why wouldn't this `cluster` argument be in the criteria?

Reply:
Thank you Jay, it seems separate param remains because the Criteria.cluster was added later, but there is a task to cleanup this code sections: https://github.com/kiali/kiali/issues/6052